apis/python/src/tiledbsoma/_multiscale_image.py

```diff
@@ -151,6 +151,11 @@ def create(
                 axis is provided, this defaults to the channel axis followed by the
                 coordinate space axes in reverse order (e.g.
                 ``("soma_channel", "y", "x")`` if ``coordinate_space=("x", "y")``).
+            has_channel_axis: Save the image with a dedicated "channel" axis.
```

Comment:
@jp-dark mentioned the `has_channel_axis` arg may go away. This can be sequenced before or after that, afaic.